src/store/lib/useOptimisticDetailsStore.js

```diff
@@ -0,0 +1,38 @@
+import { useCallback } from 'react';
+import { Just, Nothing } from 'folktale/maybe';
+
+import useSetState from 'lib/useSetState';
+
+const DEFAULT_OPTIMISTIC_DETAILS = {
+  keyRevisionNumber: Nothing(),
```

Comment:
Hmm, not sure if this is still needed, if we always make the user wait for transaction confirm. If we assume we're fully synchronous with the blockchain, we can get rid of all optimistic logic entirely.

Reply:
that's true, we can just make them wait and then just refetch the point details etc. I prefer doing that, since getting into optimistic state is a super headache. can we confirm with alex/jimmy that blocking is a-ok?

Reply:
As per email: confirmed that's a-ok!